.github/workflows/main.yml

```diff
@@ -41,7 +41,7 @@ jobs:
           path: 'test-resources/**/screenshots/*.png'
           retention-days: 2
           if-no-files-found: warn
-      - uses: codecov/codecov-action@v1
+      - uses: codecov/codecov-action@29386c70ef20e286228c72b668a06fd0e8399192 #v1
```

Comment:
This is technically [`v1.5.2`](https://github.com/codecov/codecov-action/commit/29386c70ef20e286228c72b668a06fd0e8399192).